changelogs/unreleased/gh-5843-fiber-wakeup.md

```diff
@@ -0,0 +1,6 @@
+## bugfix/core
+
+Remove assertion raised in case of `fiber_wkeup()` get called with
+dead fibers. Due to backward compatibility we allow such calls for
```

Comment:
not sure I follow you here. "we allow such calls", it is present tense. Do you prefer perfect tense instead? "Due to backward compatibility we've allowed such calls..."?

Reply:
How about this?

```
* Remove assertion raised in case of `fiber_wakeup()` get called with
  dead fibers. Due to backward compatibility we've allowed such calls
  for release builds but didn't not in debug builds. In result there
  was inconsistency between program behaviour (gh-5843).
```

Reply:
Force pushed an update